bin/packages/config.cfg.sample

```diff
@@ -266,3 +266,5 @@ default_crawler_closespider_pagecount = 50
 default_crawler_user_agent = Mozilla/5.0 (Windows NT 6.1; rv:60.0) Gecko/20100101 Firefox/60.0
 splash_url = http://127.0.0.1
 splash_port = 8050-8052
+[IP]
+networks = 192.168.34.0/24,10.0.0.0/8,192.168.33.0/24
```

Comment:
I would create one empty and comment out this line so that user do not receive alert if they don't update it but still see an example on how you can register IPs.

Reply:
I updated the PR.